blog-posts/guardian-bundle-size/guardian-bundle-size.md

```diff
@@ -0,0 +1,64 @@
+---
+published: false
+title: 'Best practice (1): control the size of your Lambda function bundle'
```

Comment:
Un truc un peu plus punchy, comme "AWS Lambda 101: shave that bundle down", qu'est ce que t'en penses ?

Reply:
Et en anglais tu capitalize tous les premières lettres : https://www.grammarly.com/blog/capitalization-in-the-titles/

Reply:
D'accord avec @fredericbarthelet !
Après, si tu veux garder quelque chose du style `Best practice (1)`, je pense qu'il vaudrait mieux transformer ça en `Serverless Best Practices (1)`

Reply:
Fixed